src/main/java/io/conduit/DestinationStream.java

```diff
@@ -50,15 +49,15 @@ public DestinationStream(SinkTask task,
 
     @Override
     public void onNext(Destination.Run.Request request) {
-        log.debug("Writing record...");
+        Logger.get().info("Writing record...");
         try {
             // Currently, Conduit requires all writes to be asynchronous.
             // See: pkg/connector/destination.go, method Write().
             Record record = request.getRecord();
             doWrite(record);
             responseObserver.onNext(responseWith(record.getPosition()));
         } catch (Exception e) {
-            log.error("Couldn't write record.", e);
+            Logger.get().error("Couldn't write record.", e);
```

Comment:
I think all logs are treated the same, no matter if they are emitted to stderr or stdout. For example the connector SDK is emitting everything to [stderr](https://github.com/ConduitIO/conduit-connector-sdk/blob/24d4e85a62f51a3f0e04fab248544d5d8b4f2a6f/logger.go#L41), the way it achieves leveled and structured logging is by creating JSON strings and setting some specific fields (e.g. `@level`, `@caller`, `@message`, `@timestamp`). Specifics can be [seen here](https://github.com/hashicorp/go-hclog/blob/b6b55671f4e5b82443139ee3e9f4417603c4cd72/intlogger.go#L501-L537).

That said, adding a proper log format can be added separately, logs will currently already be included in Conduit but with an unknown level.

Reply:
FTR, leveled logging: https://github.com/ConduitIO/conduit-kafka-connect-wrapper/issues/48